libpod/image/image.go

```diff
@@ -772,3 +781,18 @@ func splitString(input string) string {
 func (i *Image) InputIsID() bool {
 	return strings.HasPrefix(i.ID(), i.InputName)
 }
+
+// Containers returns the storage containers associated with the image
+func (i *Image) Containers() ([]storage.Container, error) {
```

Comment:
Can you change this to return []string with just the container IDs instead? I don't think Varlink needs anything more than a number of containers.

Reply:
I'd still prefer not to return storage.Container - the ID should be plenty if you want to work with the container in libpod.